README.md

```diff
@@ -1,2 +1,115 @@
 # lsp-deploy
-A place to store the logic and configuration needed to spin up an entire LSP.
+
+This is the repository to store helper scripts and per-environment
+deployment configuration for the services of the Rubin Science
+Platform.
+
+# Background
```

Comment:
I recommend h2-level headers for subsections.

Reply:
Sounds good to me!